src/lib/components/view.ts

```diff
@@ -17,10 +17,19 @@ export function View(name: ViewName): ViewImpl {
 
 export class ViewImpl extends ViewBaseImpl {
   // Add here custom implementations
-  find<T extends Data>(coll: Class<T>, query: Query, projection?: { [k: string]: unknown }): this
-  find(coll: string, query: Query, projection?: { [k: string]: unknown }): this
+  find<T extends Data>(coll: Class<T>, query: Query, projection?: { [k: string]: unknown }, options?: {
+    limit?: number | null;
+    skip?: number | null;
+  }): this
+  find(coll: string, query: Query, projection?: { [k: string]: unknown }, options?: {
+    limit?: number | null;
+    skip?: number | null;
+  }): this
```

Comment:
Use the Options interface (need to be imported), it will make it easier to use and upgrade:

```suggestion
  find<T extends Data>(coll: Class<T>, query: Query, projection?: { [k: string]: unknown }, options?: Options): this
  find(coll: string, query: Query, projection?: { [k: string]: unknown }, options?: Options): this
```